app/serializers/app.js

```diff
@@ -1,6 +1,15 @@
 import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
+  normalize() {
```

Comment:
@rwjblue -- Agreed on getting Ember Data updated.  Dashboard is loaded with redundant requests where the model should already exist in Store, but is loaded fresh from server anyways.

Should we expand this change to the entire `ApplicationSerializer` so that is transposes all `belongsTo` directly onto the payload?  I'm experimenting with the following in my ApplicationSerializer:

```javascript
  normalize: function(type, hash, property) {
    var payload = this._super(...arguments);

    if (payload.links && payload.links.account) {
      payload.links.stack = payload.links.account;
      delete payload.links.account;
    }
    if (payload.links && payload.links.accounts) {
      payload.links.stacks = payload.links.accounts;
      delete payload.links.accounts;
    }

    if(payload.links) {
      type.eachRelationship((modelName, modelClass) => {
        if(!modelClass.kind !== 'belongsTo') {
          return;
        }

        let relationHref = payload.links[modelName];

        if(relationHref && !payload[modelName]) {
          payload[modelName] = this._idFromHref(relationHref);
        }
      });
    }

    return payload;
  },
```

This would be removed after updating ET and HAL9000.